packages/local_database_flutter/README.md

```diff
@@ -1,6 +1,6 @@
-# repository
+# local_database
```

Comment:
why `local_database`? just wondering what level of abstraction it will have. if it is dependent on local sqlite, does it actually make sense to extract it as a separate package? 
this is not a blocker.

Reply:
This is done in order to separate the data and operations on them from their presentation in flutter app.
It can depend on local sqlite or on anything else, but it must seperate any actions over data from main program package.
Also, it is required by sepereated buissness logic patterns (blocs).